security-analysis/security-analysis-api/src/main/java/com/powsybl/security/monitor/StateMonitor.java

```diff
@@ -20,19 +20,14 @@
 import java.util.*;
 
 /**
+ *  A state monitor allows to output in security analysis results some state variables related to branches, buses and three
+ *  windings transformers. The supported state variables are active power, reactive power and current at both side for branches
+ *  (see {@link com.powsybl.security.results.BranchResult}), active power, reactive power and current at network side for
+ *  three windings transformers (see {@link com.powsybl.security.results.ThreeWindingsTransformerResult}) and voltage angle and voltage
+ *  magnitude for buses (see {@link com.powsybl.security.results.BusResult}). A state monitor is defined in a contingency context.
```

Comment:
Rather `defined for a specific`? `defined in` seems to mean that ContingencyContext contains a `StateMonitor` parameter.

We could also add that, depending on the context, the corresponding results are available in `SecurityAnalysisResult.getPreContingencyResult()` or among the `SecurityAnalysisResult.getPostContingencyResults()`.

Reply:
A state monitor is defined for some contingencies through a {@link ContingencyContext}.

Reply:
A NONE or ALL (because it includes the base case) contingency context will output some monitoring results in {@link PreContingencyResult}. If a contingency has a state monitor declared (through a ALL, a ONLY_CONTINGENCY or a SPECIFIC), monitoring results are output in the dedicated {@link PostContingencyResult}.